examples/http_download.json

```diff
@@ -4,12 +4,22 @@
 		{
 			"id": "source_path",
 			"type" : "string",
-			"value" : "https://media-io.com"
+			"value" : "index.html"
```

Comment:
is it possible to set:
`"value": "http://localhost:8080/index.html"`
and in the `source_secret` ship the endpoint parameter ?

Lot of time the URL is a full URL, it will be complex to manage the split ...

Reply:
the question is also for query parameters like:
`"value": "http://localhost:8080/index.html?token=123456"`